op-batcher/batcher/driver.go

```diff
@@ -256,7 +256,11 @@ func (l *BatchSubmitter) loop() {
 			if err := l.loadBlocksIntoState(l.shutdownCtx); errors.Is(err, ErrReorg) {
 				err := l.state.Close()
 				if err != nil {
-					l.Log.Error("error closing the channel manager to handle a L2 reorg", "err", err)
+					if errors.Is(err, ErrPendingAfterClose) {
+						l.Log.Warn("closed channel manager to handle L2 reorg, but pending channel(s) remain to be submitted")
```

Comment:
I think we can downgrade this message to `Info` because this is an expected situation. Same below.

Reply:
It's an expected situation, but still risky w.r.t. unclean shutdowns, so I think it's worth a warn-log.